### ISSUE TITLE
Win10系统下，分类套件训练报错

### ISSUE THREAD

**Issue by kobe577590** [PaddlePaddle/PaddleClas#632]
环境系统：
win10(64位)+NVIDIA CUDA (11.0.228)

运行预训练模型进行体验30分钟套件分类体验项目，发现无法进行正常训练，操作如下：
`python -m paddle.distributed.launch --gpus="0" tools/train.py  -c ./configs/quick_start/ResNet50_vd_finetune.yaml`

日志如下：

`C:\Users\Lenovo\anaconda3\lib\site-packages\win32\lib\pywintypes.py:2: DeprecationWarning: the imp module is deprecated in favour of importlib; see the module's documentation for alternative uses     
  import imp, sys, os
-----------  Configuration Arguments -----------
gpus: 0
heter_worker_num: None
heter_workers: 
http_port: None
ips: 127.0.0.1
log_dir: log
nproc_per_node: None
server_num: None
servers: 
training_script: tools/train.py
training_script_args: ['-c', './configs/quick_start/ResNet50_vd_finetune.yaml']
worker_num: None
workers: 
------------------------------------------------
WARNING 2021-03-11 09:37:55,026 launch.py:316] Not found distinct arguments and compiled with cuda. Default use collective mode
launch train in GPU mode
INFO 2021-03-11 09:37:55,031 launch_utils.py:471] Local start 1 processes. First process distributed environment info (Only For Debug): 
    +=======================================================================================+
    |                        Distributed Envs                      Value                    |
    +---------------------------------------------------------------------------------------+
    |                       PADDLE_TRAINER_ID                        0                      |
    |                 PADDLE_CURRENT_ENDPOINT                 127.0.0.1:56707               |
    |                     PADDLE_TRAINERS_NUM                        1                      |
    |                PADDLE_TRAINER_ENDPOINTS                 127.0.0.1:56707               |
    |                     FLAGS_selected_gpus                        0                      |
    +=======================================================================================+

INFO 2021-03-11 09:37:55,033 launch_utils.py:475] details abouts PADDLE_TRAINER_ENDPOINTS can be found in log/endpoints.log, and detail running logs maybe found in log/workerlog.0
子目录或文件 -p 已经存在。
处理: -p 时出错。
子目录或文件 log 已经存在。
处理: log 时出错。
'rm' 不是内部或外部命令，也不是可运行的程序
或批处理文件。
C:\Users\Lenovo\anaconda3\lib\site-packages\win32\lib\pywintypes.py:2: DeprecationWarning: the imp module is deprecated in favour of importlib; see the module's documentation for alternative uses
  import imp, sys, os
2021-03-11 09:37:58,548 - INFO - 
===========================================================
==        PaddleClas is powered by PaddlePaddle !        ==
===========================================================
==                                                       ==
==   For more info please go to the following website.   ==
==                                                       ==
==       https://github.com/PaddlePaddle/PaddleClas      ==
===========================================================

2021-03-11 09:37:58,549 - INFO - ARCHITECTURE :
2021-03-11 09:37:58,549 - INFO -     name : ResNet50_vd
2021-03-11 09:37:58,549 - INFO - ------------------------------------------------------------
2021-03-11 09:37:58,549 - INFO - LEARNING_RATE :
2021-03-11 09:37:58,549 - INFO -     function : Cosine
2021-03-11 09:37:58,549 - INFO -     params :
2021-03-11 09:37:58,549 - INFO -         lr : 0.00375
2021-03-11 09:37:58,549 - INFO - ------------------------------------------------------------
2021-03-11 09:37:58,549 - INFO - OPTIMIZER :
2021-03-11 09:37:58,549 - INFO -     function : Momentum
2021-03-11 09:37:58,550 - INFO -     params :
2021-03-11 09:37:58,550 - INFO -         momentum : 0.9
2021-03-11 09:37:58,550 - INFO -     regularizer :
2021-03-11 09:37:58,550 - INFO -         factor : 1e-06
2021-03-11 09:37:58,550 - INFO -         function : L2
2021-03-11 09:37:58,550 - INFO - ------------------------------------------------------------
2021-03-11 09:37:58,550 - INFO - TRAIN :
2021-03-11 09:37:58,550 - INFO -     batch_size : 32
2021-03-11 09:37:58,550 - INFO -     data_dir : ./dataset/flowers102/
2021-03-11 09:37:58,550 - INFO -     file_list : ./dataset/flowers102/train_list.txt
2021-03-11 09:37:58,550 - INFO -     num_workers : 0
2021-03-11 09:37:58,550 - INFO -     shuffle_seed : 0
2021-03-11 09:37:58,550 - INFO -     transforms : 
2021-03-11 09:37:58,550 - INFO -         DecodeImage :
2021-03-11 09:37:58,550 - INFO -             channel_first : False
2021-03-11 09:37:58,550 - INFO -             to_np : False
2021-03-11 09:37:58,551 - INFO -             to_rgb : True
2021-03-11 09:37:58,551 - INFO -         RandCropImage :
2021-03-11 09:37:58,551 - INFO -             size : 224
2021-03-11 09:37:58,551 - INFO -         RandFlipImage :
2021-03-11 09:37:58,551 - INFO -             flip_code : 1
2021-03-11 09:37:58,551 - INFO -         NormalizeImage :
2021-03-11 09:37:58,551 - INFO -             mean : [0.485, 0.456, 0.406]
2021-03-11 09:37:58,551 - INFO -             order : 
2021-03-11 09:37:58,551 - INFO -             scale : 1./255.
2021-03-11 09:37:58,551 - INFO -             std : [0.229, 0.224, 0.225]
2021-03-11 09:37:58,551 - INFO -         ToCHWImage : None
2021-03-11 09:37:58,551 - INFO - ------------------------------------------------------------
2021-03-11 09:37:58,551 - INFO - VALID :
2021-03-11 09:37:58,551 - INFO -     batch_size : 20
2021-03-11 09:37:58,551 - INFO -     data_dir : ./dataset/flowers102/
2021-03-11 09:37:58,551 - INFO -     file_list : ./dataset/flowers102/val_list.txt
2021-03-11 09:37:58,552 - INFO -     num_workers : 0
2021-03-11 09:37:58,552 - INFO -     shuffle_seed : 0
2021-03-11 09:37:58,552 - INFO -     transforms :
2021-03-11 09:37:58,552 - INFO -         DecodeImage : 
2021-03-11 09:37:58,552 - INFO -             channel_first : False
2021-03-11 09:37:58,552 - INFO -             to_np : False
2021-03-11 09:37:58,552 - INFO -             to_rgb : True
2021-03-11 09:37:58,552 - INFO -         ResizeImage :
2021-03-11 09:37:58,552 - INFO -             resize_short : 256
2021-03-11 09:37:58,552 - INFO -         CropImage :
2021-03-11 09:37:58,552 - INFO -             size : 224
2021-03-11 09:37:58,552 - INFO -         NormalizeImage :
2021-03-11 09:37:58,552 - INFO -             mean : [0.485, 0.456, 0.406]
2021-03-11 09:37:58,552 - INFO -             order :
2021-03-11 09:37:58,552 - INFO -             scale : 1.0/255.0
2021-03-11 09:37:58,553 - INFO -             std : [0.229, 0.224, 0.225]
2021-03-11 09:37:58,553 - INFO -         ToCHWImage : None
2021-03-11 09:37:58,553 - INFO - ------------------------------------------------------------
2021-03-11 09:37:58,553 - INFO - classes_num : 102
2021-03-11 09:37:58,553 - INFO - epochs : 20
2021-03-11 09:37:58,553 - INFO - image_shape : [3, 224, 224]
2021-03-11 09:37:58,553 - INFO - mode : train
2021-03-11 09:37:58,553 - INFO - model_save_dir : ./output/
2021-03-11 09:37:58,553 - INFO - pretrained_model : ./pretrained/ResNet50_vd_pretrained
2021-03-11 09:37:58,553 - INFO - save_interval : 1
2021-03-11 09:37:58,553 - INFO - topk : 5
2021-03-11 09:37:58,553 - INFO - total_images : 1020
2021-03-11 09:37:58,553 - INFO - valid_interval : 1
2021-03-11 09:37:58,553 - INFO - validate : True
C:\Users\Lenovo\anaconda3\lib\site-packages\paddle\fluid\dygraph\layers.py:1303: UserWarning: Skip loading for out.weight. out.weight receives a shape [2048, 1000], but the expected shape is [2048, 102].
  warnings.warn(("Skip loading for {}. ".format(key) + str(err)))
C:\Users\Lenovo\anaconda3\lib\site-packages\paddle\fluid\dygraph\layers.py:1303: UserWarning: Skip loading for out.bias. out.bias receives a shape [1000], but the expected shape is [102].
  warnings.warn(("Skip loading for {}. ".format(key) + str(err)))
2021-03-11 09:38:05,524 - INFO - Finish initing model from ./pretrained/ResNet50_vd_pretrained
INFO 2021-03-11 09:39:10,236 launch_utils.py:307] terminate all the procs
ERROR 2021-03-11 09:39:10,242 launch_utils.py:545] ABORT!!! Out of all 1 trainers, the trainer process with rank=[0] was aborted. Please check its log.
INFO 2021-03-11 09:39:13,243 launch_utils.py:307] terminate all the procs`

**发现无明显报错，只是出现如下信息：**
- 【terminate all the procs 】 
- 【ABORT!!! Out of all 1 trainers, the trainer process with rank=[0] was aborted. Please check its log.】
-  【terminate all the procs】
**期待各位大神答疑，指点迷津，被卡了两天了**

**Comment by TingquanGao** [comment]
请尝试将启动命令改为：`python tools/train.py -c ./configs/quick_start/ResNet50_vd_finetune.yaml`
